R/SouthAfrica.R

```diff
@@ -0,0 +1,85 @@
+#' SouthAfrica Class for downloading, cleaning and processing notification data
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for South Africa.
+#'
+#' @details Inherits from `DataClass`
+#' @source https://raw.githubusercontent.com/dsfsi/covid19za/master/data/covid19za_provincial_cumulative_timeline_confirmed.csv # nolint
+#' @source https://raw.githubusercontent.com/dsfsi/covid19za/master/data/covid19za_provincial_cumulative_timeline_deaths.csv # nolint
+#' @export
+#' @examples
+#' \dontrun{
+#' region <- SouthAfrica$new(verbose = TRUE, steps = TRUE, get = TRUE)
+#' region$return()
+#' }
+SouthAfrica <- R6::R6Class("SouthAfrica",
+  inherit = DataClass,
+  public = list(
+    # Core Attributes (amend each paramater for country specific infomation)
```

Comment:
paramater -> parameter
infomation -> information 

...

Honestly, just delete the parenthetical comment - I think this is a holdover from a template.